app.py

```diff
@@ -285,11 +285,16 @@ def parse():
                 response["Code"] = "# coding=utf8\n" + E.arguments['fixed_code']
                 response["Warning"] = error_template.format(**E.arguments)
             elif E.args[0] == "Parse":
-                error_template = hedy_errors[E.error_code]
-                # Localize the names of characters
-                if 'character_found' in E.arguments:
-                    E.arguments['character_found'] = hedy_errors[E.arguments['character_found']]
-                response["Error"] = error_template.format(**E.arguments)
+                try:
+                    error_template = hedy_errors[E.error_code]
+                    # Localize the names of characters
+                    if 'character_found' in E.arguments:
+                        E.arguments['character_found'] = hedy_errors[E.arguments['character_found']]
+                    response["Error"] = error_template.format(**E.arguments)
+                except:
+                    print(code) #TEMP!! We need to find the programs that cause weird issue #375
```

Comment:
Could you add a prefix here to the `print` saying `DEBUG ISSUE 375` (or something of the like) so we can easily identify it? Something like `print ('DEBUG ISSUE 375', code)`

Reply:
> print ('DEBUG ISSUE 375', code)

Good idea, added!